index.js

```diff
@@ -1,12 +1,68 @@
-const mongoose = require('mongoose');
-const Recipe = require('./models/Recipe'); // Import of the model Recipe from './models/Recipe'
-const data = require('./data.js');  // Import of the data from './data.js'
+const mongoose = require("mongoose");
+const Recipe = require("./models/Recipe"); // Import of the model Recipe from './models/Recipe'
+const data = require("./data.js"); // Import of the data from './data.js'
 
 // Connection to the database "recipeApp"
-mongoose.connect('mongodb://localhost/recipeApp', { useNewUrlParser: true })
+mongoose
+  .connect("mongodb://localhost/recipeApp", { useNewUrlParser: true })
   .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
+    console.log("Connected to Mongo!");
+  })
+  .catch(err => {
+    console.error("Error connecting to mongo", err);
   });
 
+Recipe.create({
+  title: "Cake",
+  level: "Easy Peasy",
+  ingredients: "strawberry",
+  cuisine: "Turkish",
+  dishType: "Dessert",
+  image: "image",
+  duration: 40,
+  creator: "Yagmur",
+  created: "07.07.2019"
+});
```

Comment:
You can add a promise at the end, just to get the feedback 

such as 

`  .then(recipeDoc => {
    console.log("New recipe CREATED correctly ! Yummmm.", recipeDoc);
  })
  .catch(err => {
    console.log("Recipe's creation failed...", err);
  });`